src/Abp/Authorization/AuthorizeAttributeHelper.cs

```diff
@@ -16,11 +19,13 @@ public AuthorizeAttributeHelper()
         {
             AbpSession = NullAbpSession.Instance;
             PermissionChecker = NullPermissionChecker.Instance;
+            MultiTenancyConfig = new MultiTenancyConfig();
```

Comment:
It is property injected, Didn't see a NullMultitenancyConfig, so I just created a new one.